examples/vmod_object/README.md

```diff
@@ -19,19 +19,12 @@ import object;
 import object from "path/to/libobject.so";
 ```
 
-### Object `kv`
-
-```vcl
-// Create a new instance of the object in your VCL init function
-sub vcl_init {
-    new new = kv.new([INT cap]);
-}
-```
+### Constructor `object.kv([INT cap])`
 
 Create a new key-value store, with an optional capacity.
 If `cap` is 0 or less, it will be ignored.
 
-#### Method `STRING get(STRING key)`
+#### Method `STRING .get(STRING key)`
```

Comment:
that's not right...

Reply:
* `<instance_name>.get(...)` ?
* `<variable>.get(...)`
* `<var>.get(...)`
* `<obj>.get(...)`

But i do see your point here

Reply:
I think I like `<obj>`, it's concise and it will catch your eye and force you to think. I'll go with that as a test

Reply:
perhaps `<object>` to be even more precises?

Reply:
(acronyms are evil in docs ;) )

Reply:
eh, you were the one offering!

Reply:
I'm in an AI "reasoning" mode - first i spew out lots of raw ideas, and then go back and review them and give a better answer :-P